### PR TITLE
fix(northlight/landing): Giants block causing page size issues

### PR DESCRIPTION
**Pull request by ilija-**
This fix is resolving the giants block going out of the viewport issue, and causing the appearance of a horizontal scroll on the /components page after paolo's card was added.

### CODE REVIEW

File: framework/sandbox/docs/app.tsx [hunk 1]
```diff
@@ -265,11 +265,10 @@ const DocsApp = () => {
         </Fade>
       </Flex>
 
-      <Flex minWidth="max-content" minHeight={ 80 } alignItems="center" mt={ 24 } justifyContent="center" gap="2">
         <Fade in={ true } enterDelay={ 800 }>
           <H1 mb={ 5 }>Built by Giants</H1>
-          <HStack borderColor="white" boxShadow="none">
-            <Card maxW="xs" boxShadow="none" borderColor="white" bgColor="transparent">
+          <SimpleGrid spacing={4} templateColumns='repeat(auto-fit, minmax(220px, 1fr))'>
+          <Card maxW="xs" boxShadow="none" borderColor="white" bgColor="transparent">
```
Comment (magnusohlin) [inline framework/sandbox/docs/app.tsx:271]: Indent this row

File: framework/sandbox/docs/app.tsx [hunk 2]
```diff
@@ -403,10 +402,9 @@ const DocsApp = () => {
                 </ButtonGroup>
               </CardFooter>
             </Card>
-          </HStack>
-
+
```
Comment (magnusohlin) [inline framework/sandbox/docs/app.tsx:405]: Remove blank row

**Review by magnusohlin — CHANGES_REQUESTED** [review]
Just some linting errors

**Comment by SebCodesTheWeb** [comment]
This is not how you want to name your commits, always follow conventional commits: https://www.conventionalcommits.org/en/v1.0.0/. And if a fix was needed, like linting, you want to amend it, or squash it to the previous commit, basically combining them. You can ask me if you need help with this.

**Comment by magnusohlin** [comment]
https://user-images.githubusercontent.com/521837/221791480-5810f55c-d564-4451-a9fa-54534419e6ad.mov

@ilija- I think you have received the branch-rebase script from me, I did a little video for you on how to squash your commits

**Comment by magnusohlin** [comment]
There is a slight error in my video above, the top commit should be fix(framework): bla bla

**Review by magnusohlin — APPROVED** [review]

**Pull request merged**